call-notes/call_6.md

```diff
@@ -0,0 +1,160 @@
+
+So, there are a lot of problems with this current restart process. Maybe the biggest problem is it takes a long time. So the whole cluster restart takes about, it could take about like several hours. Which would make your reliability not very good. And so, in the future, we might have other efforts to make it faster, but today we are just trying to solve a small problem. And so we are trying to improve the process of finding the highest optimistically confirmed block because it is really hard for human to pull like two thousand to three thousand validators and see what each one voted for, and then decide what the block is. This is really a job better for machine. So, we are trying to see whether we can develop a protocol, so that the machines can automatically find the highest confirmed block without human intervention.
+
+So, the main design goal here is not to have `false negatives` that means `if a block was confirmed before the restart` that is why not roll it back because that will have very bad impact. And, but `false positives` might be okay. It means maybe some block is not confirmed before the restart but somehow we think it is confirmed which is fine. Let's say there's our limit is at `67`. Let's say there is a block which got `66%` of the votes and there are no competing blocks. So this is the only one we know which got 66 percent of the votes. So it is okay if we the 80% of the validators in restart decide that we should start from here even though it is not confirmed. It is okay to confirm more, but it's not okay to roll back confirmed block. So, in all the choices we have after this, we will see that we try to not have `false negatives` at all. And `false positives`? that might be fine in the design choices. So, the new approach we think because we now want to have machines automatically negotiate what is the block we want to restart that from. So we would still have humans in the room, they would still start restart validators with the argument. Let's say optimists restart, but they don't need to tell you which slot and which hash they need to start from. That's for machines to decide and we would add additional phase, the silent repair phase here is where the validators negotiate with themselves what is a block they should restart from. So, because they are in this repair phase, no one would make new blocks anymore. So there would be no [turbine](https://docs.solana.com/cluster/turbine-block-propagation) traffic at all. The goal here is to just converge as soon as possible.
```

Comment:
```suggestion
So, the main design goal here is not to have false negatives. That means if a block was confirmed before the restart that is why not roll it back because that will have very bad impact. And, but false positives might be okay. It means maybe some block is not confirmed before the restart but somehow we think it is confirmed which is fine. Let's say there's our limit is at 67. Let's say there is a block which got 66% of the votes and there are no competing blocks. So this is the only one we know which got 66% of the votes. So it is okay if 80% of the validators in restart decide that we should start from here even though it is not confirmed. It is okay to confirm more, but it's not okay to roll back confirmed block. So, in all the choices we have after this, we will see that we try to not have false negatives at all. And false positives might be fine in the design choices. So, the new approach we think because we now want to have machines automatically negotiate what is the block we want to restart that from. So we would still have humans in the room, they would still start restart validators with the argument. Let's say optimists restart, but they don't need to tell you which slot and which hash they need to start from. That's for machines to decide and we would add additional phase, the silent repair phase here is where the validators negotiate with themselves what is a block they should restart from. So, because they are in this repair phase, no one would make new blocks anymore. So there would be no [turbine](https://docs.solana.com/cluster/turbine-block-propagation) traffic at all. The goal here is to just converge as soon as possible.
```